aws-sso-instanceaccesscontrolattributeconfiguration/aws-sso-instanceaccesscontrolattributeconfiguration.json

```diff
@@ -0,0 +1,117 @@
+{
+  "typeName": "AWS::SSO::InstanceAccessControlAttributeConfiguration",
+  "description": "Resource Type definition for SSO InstanceAccessControlAttributeConfiguration",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-sso/aws-sso-instanceaccesscontrolattributeconfiguration",
+  "definitions": {
+    "AccessControlAttributeValueSource": {
+      "type": "string",
+      "minLength": 0,
+      "maxLength": 256,
+      "pattern": "[\\p{L}\\p{Z}\\p{N}_.:\\/=+\\-@\\[\\]\\{\\}\\$\\\\\"]*"
+    },
+    "AccessControlAttributeValueSourceList": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/AccessControlAttributeValueSource"
+      },
+      "maxItems": 1
+    },
+    "AccessControlAttributeValue": {
+      "type": "object",
+      "properties": {
+        "Source": {
+          "$ref": "#/definitions/AccessControlAttributeValueSourceList"
+        }
+      },
+      "required": [
+        "Source"
+      ],
+      "additionalProperties": false
+    },
+    "AccessControlAttribute": {
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "pattern": "[\\p{L}\\p{Z}\\p{N}_.:\\/=+\\-@]+",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "$ref": "#/definitions/AccessControlAttributeValue"
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    },
+    "AccessControlAttributeList": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/AccessControlAttribute"
+      },
+      "maxItems": 50
+    }
+  },
+  "properties": {
+    "InstanceArn": {
+      "description": "The sso instance that the InstanceAccessControlAttributeConfiguration will be owned.",
+      "type": "string",
+      "pattern": "arn:aws:sso:::instance/(sso)?ins-[a-zA-Z0-9-.]{16}",
```

Comment:
[should un-hardcode the `aws` partition](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arns-syntax) eventually in order to work in [other partitions](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants)